.gitignore

```diff
@@ -45,5 +49,7 @@ README.html
 
 sandbox
 
+# VSCode
+.vscode/
```

Comment:
Editor-specific files belong in global gitignore files (https://help.github.com/en/articles/ignoring-files#create-a-global-gitignore) instead of repo gitignores

Reply:
Nice catch. Fixed.